testing/rust/need-rpath.patch

```diff
@@ -0,0 +1,49 @@
+--- a/src/librustc_back/target/mod.rs
```

Comment:
Please add patch header and description – what this patch actually do and why it’s needed. You can probably just copy description from `force-rpath-on-prefer-dynamic.patch`…?